extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetTableWriter.java

```diff
@@ -648,6 +668,14 @@ private static int getTargetSize(Class<?> columnType) throws IllegalAccessExcept
             return new ByteTransfer(columnSource, targetSize);
         } else if (String.class.equals(columnType)) {
             return new StringTransfer(columnSource, targetSize);
+        } else if (BigDecimal.class.equals(columnType)) {
+            // noinspection unchecked
+            final ColumnSource<BigDecimal> bigDecimalColumnSource = (ColumnSource<BigDecimal>) columnSource;
+            final TypeInfos.PrecisionAndScale precisionAndScale = TypeInfos.getPrecisionAndScale(
```

Comment:
We can and we are?
Writing the code this way means that it doesn't matter if we rearrange it later, instead of having to know.  Whatever does something first puts it in the cache, whatever does something second finds it there.